function.dd

```diff
@@ -367,146 +367,164 @@ $(H4 $(LNAME2 inout-functions, Inout Functions))
     )
 
 
-$(H4 $(LNAME2 property-functions, Property Functions))
-
-        $(P Property functions are tagged with the $(D @property)
-        attribute. They cannot be called with parentheses (hence
-        they act like fields except in some cases) ($(B note:) dmd does not
-        enforce this currently).
-        )
+$(H4 $(LNAME2 optional-parenthesis, Optional Parentheses))
 
-        $(P If a property function has no parameters, it works as a getter.
-        If has exactly one parameter, it works as a setter.
+        $(P If a function call does not take any arguments syntactically,
+        it is callable without parentheses, like a getter $(RELATIVE_LINK2 property-functions, property function).
```

Comment:
I get what you're saying here, i.e. include cases like default arguments etc. Here's a more direct way of saying it: replace

"If a function call does not take any arguments syntactically, it is callable without parentheses, like a getter."

with

"If a function call passes no explicit argument, i.e. it would syntactically use $(D ()), then these parentheses may be omitted, similar to a getter invocation."